colorfield/static/colorfield/colorfield.js

```diff
@@ -1,12 +1,12 @@
 /** global: django */
 
-window.addEventListener('load', function(event) {
-    if (typeof(django) !== 'undefined' && typeof(django.jQuery) !== 'undefined') {
-        (function($) {
-            // add colopicker to inlines added dynamically
-            $(document).on('formset:added', function onFormsetAdded(event, row) {
-                jscolor.installByClassName('jscolor');
-            });
-        })(django.jQuery);
-    }
+window.addEventListener("load", function (event) {
+  if (typeof django !== "undefined" && typeof django.jQuery !== "undefined") {
+    (function ($) {
+      // add colopicker to inlines added dynamically
+      $(document).on("formset:added", function onFormsetAdded(event, row) {
+        jscolor.install();
```

Comment:
Have you tested it with inlines?

Reply:
✔️ 
Yep. Here's a demo  https://recordit.co/0E6QNLhU5b

Reply:
Good! I see that you use also `django-admin-interface` :fist_right::fist_left: